### DISCUSSION TITLE
[V3] Docs typo

### DISCUSSION THREAD

**Discussion by m7moudabdel7mid** [livewire/livewire#6152] [4. General]
Hi,

There is an error in the docs on Validation page > Using custom validators
in the code samples the imported Validator class should be `Illuminate\Support\Facades\Validator` since `Illuminate\Validation\Validator` cause an error `Call to undefined method Illuminate\Validation\Validator::make()`.

**Comment by joshhanley** [comment]
@m7moudabdel7mid thanks for reporting! PRs to fix the docs are welcome! 🙂
